bundles/org.openhab.binding.bambulab/README.md

```diff
@@ -0,0 +1,172 @@
+While cloud mode is theoretically possible, it is not supported by the addon developers.
+
+## Cloud Mode
+
+Cloud mode is possible but not officially supported by the addon developers.
+
+To use cloud mode, follow these steps:
+
+### Find Username
+
+Log in to Maker World and visit [my-preferences](https://makerworld.com/api/v1/design-user-service/my/preference) to retrieve a JSON response containing your data. 
+The relevant field is `uid`, which represents the unique ID of your account. 
+Use this value as the `username` in the configuration (advanced field) with the prefix `u_`.
+
+### Access Token
+
+To obtain an access token, follow these steps:
+
+1. Log in using your email and password.
+2. Confirm the login using a token received via email.
+
+#### Step 1: Login with Email and Password
+
+```shell
+curl -X POST "https://api.bambulab.com/v1/user-service/user/login" \
+     -H "Content-Type: application/json" \
+     -d '{
+           "account": "you@email.io",
+           "password": "superduperpassword123"
+         }'
+```
+
+#### Step 2: Confirm Login with Token from Email
+
+```shell
+curl -X POST "https://api.bambulab.com/v1/user-service/user/login" \
+     -H "Content-Type: application/json" \
+     -d '{
+           "account": "you@email.io",
+           "code": "123456"
+         }'
+```
+
+You will receive a long access code in the response. Copy it and use it as the `accessCode` parameter.
+
+**Note:** This access code expires after three months. When it expires, repeat the process to obtain a new one.
+
+### Hostname
+
+Use `us.mqtt.bambulab.com` as the hostname.
+
+## Supported Things
+
+- `printer`: Represents a BambuLab 3D printer.
+
+## Thing Configuration
+
+| Parameter    | Type    | Required | Description                                                                                     |
+|--------------|---------|----------|-------------------------------------------------------------------------------------------------|
+| `serial`     | Text    | Yes      | Unique serial number of the printer.                                                            |
+| `scheme`     | Text    | No       | URI scheme. (Advanced)                                                                          |
+| `hostname`   | Text    | Yes      | IP address of the printer or `us.mqtt.bambulab.com` for cloud mode.                             |
+| `port`       | Integer | No       | URI port. (Advanced)                                                                            |
+| `username`   | Text    | No       | `bblp` for local mode or your Bambu Lab user (starting with `u_`). (Advanced)                   |
+| `accessCode` | Text    | Yes      | Access code for the printer. The method of obtaining this varies between local and cloud modes. |
+
+## Channels
+
+| Channel ID                | Type                | Description                                                      |
```

Comment:
Nit: could you use markdowntable.com or your IDE build in tool to align this table.

Reply:
yeah, normally Im using my IDE formatter that does that, but since we have spotless I've turned it off